airbyte-integrations/connectors/source-genesys/source_genesys/source.py

```diff
@@ -17,7 +17,9 @@ class GenesysStream(HttpStream, ABC):
     url_base = "https://api.mypurecloud.com.au/api/v2/"
     page_size = 500
 
-    def __init__(self, *args, **kwargs):
+    def __init__(self, api_base_url, *args, **kwargs):
+        if api_base_url is not None:
+            self.url_base = api_base_url + "/api/v2/"
```

Comment:
Please create a property function to resolve this.
```
@property
def url_base(self, api_base_url):
    return api_base_url + "/api/v2/"
```

Reply:
@marcosmarxm - I've updated the code, please take a look. thanks!